GHzDACs/test/test_jump_table.py

```diff
@@ -0,0 +1,108 @@
+"""
+
+This is intended to test servers.GHzDACs.jumpTable.py
+
+"""
+
+import numpy as np
+import pytest
+import servers.GHzDACs.jumpTable as jumpTable
+
+
+def test_limits():
+    with pytest.raises(ValueError):
+        jumpTable.JumpEntry(-1, 0, jumpTable.END())
+    with pytest.raises(ValueError):
+        jumpTable.JumpEntry(9999, 0, jumpTable.END())
+    with pytest.raises(ValueError):
+        jumpTable.JumpEntry(10, -1, jumpTable.END())
+    with pytest.raises(ValueError):
+        jumpTable.JumpEntry(10, 9999, jumpTable.END())
+
+
+def test_end():
+    end = jumpTable.JumpEntry(259, 0, jumpTable.END())
+    data = end.as_bytes()
+    assert data[0] == data[1] == 1
```

Comment:
It might be clearer to say something like:
```python
assert list(data) == [1,1,0,0,0,0,7,0]
```
Also, then the error message would show the full expected and actual values.